es-design-system/pages/atoms/icons.vue

```diff
@@ -376,6 +376,37 @@
             </li>
         </ul>
 
+        <h2>
+            Social Icons
+        </h2>
+        <p>
+            Default size for social icons is 31px wide by 32px tall.
```

Comment:
Nit: I'd love if we could eventually get 32px by 32px assets from design so these can match the other icons in being an exact square, but that can be done later.